downstream/modules/automation-hub/con-offline-token-active.adoc

```diff
@@ -0,0 +1,11 @@
+
+[id="con-offline-token-active"]
```

Comment:
Can we remove "con-", "ref-", and "proc-" fragments from IDs? They aren't meaningful and end up as part of the URL which isn't great for SEO.

Reply:
Yes, I'll remove it. Thanks @oraNod!